manual/installation.tex

```diff
@@ -614,7 +614,7 @@ \subsection{Installing on ORNL OLCF Titan Cray XK7 (NVIDIA GPU
 module load cray-hdf5
 module load cmake
 module load fftw
-export FFTW_HOME=$FFTW_DIR/..
+export FFTW_HOME=$(echo $FFTW_DIR | sed -e 's/\/lib//')
```

Comment:
Please explain this change. Is this causing problems? I used the build script yesterday on titan and it was fine.